.github/workflows/build.yml

```diff
@@ -1,7 +1,12 @@
 name: Build
 
 # Triggered when code is pushed to any branch in a repository or manually
```

Comment:
Is the comment still accurate?

Reply:
no it doesn't anymore since I excluded the main branch